File: javafolder/CicloWhile.java
package javafolder;

public class CicloWhile {
    public static void main(String[] args) {
        int numero = 5;
        int numero_adivinado = Integer.parseInt(System.console().readLine("Adivina el numero: "));
        
        while (numero_adivinado != numero) { 
            System.out.println("Numero incorrecto, vuelve a intentarlo");
            numero_adivinado = Integer.parseInt(System.console().readLine("Adivina el numero: "));
        }

        System.out.println("Felicidades, has adivinado el numero: " + numero);

    }
}

File: python/Sesion3/ciclo_while.py

numero = 5

numero_adivinado = int(input("Adivina el numero: "))

while(numero_adivinado != numero):
    print("Numero incorrecto, vuelve a intentarlo")
    numero_adivinado = int(input("Adivina el numero: "))

print("Has adivinado el numero")
